src/vconsole/vconsole-setup.c

```diff
@@ -282,6 +285,7 @@ int main(int argc, char **argv) {
         utf8 = is_locale_utf8();
 
         r = parse_env_file("/etc/vconsole.conf", NEWLINE,
+                           "VC_MAX", &vc_max_s,
```

Comment:
I guess the idea is to make it easier for low powered devices on which the time to this could matter to be able to easily initialize a few consoles less.

So far we used 15, but this is overkill, most people stick to the terminals which are easily accessible through the keyboard, so 1-12. This should be mentioned in the commit message, but 12 is probably a better default.